pkg/controller/ttlafterfinished/ttlafterfinished_controller.go

```diff
@@ -198,7 +198,7 @@ func (tc *Controller) processJob(key string) error {
 		return err
 	}
 
-	klog.V(4).Infof("Checking if Job %s/%s is ready for cleanup", namespace, name)
+	klog.V(4).InfoS("Checking if Job is ready for cleanup", "jobNamespace", namespace, "job", name)
```

Comment:
Same as above

Reply:
This can actually use `klog.KRef`, thanks for catching it.